src/base1/Makefile.am

```diff
@@ -23,6 +23,8 @@ basedebug_DATA = \
 	dist/base1/cockpit.min.js.map \
 	$(NULL)
 
+AM_TESTS_ENVIRONMENT = G_DEBUG=fatal-criticals
```

Comment:
according to the [docs](https://www.gnu.org/software/automake/manual/html_node/Scripts_002dbased-Testsuites.html#Simple%20tests%20and%20color-tests):

> for this to work portably, however, the contents of a non-empty AM_TESTS_ENVIRONMENT must be terminated by a semicolon